src/util/eventfilter_parser/example_queries/case_0.txt

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This should be in the /examples folder.
Also, use the CC0 license for this.
Otherwise people might wonder if they are allowed to build upon the examples you provide for their own queries.